tests/unit/htdocs/ConstructorReturnConventionTest.php

```
@@ -0,0 +1,158 @@
+                        continue;
+                    }
+                    if (';' !== $next) {
+                        $violations[] = (int) $token[2];
+                    }
+                    break;
+                }
+            }
+        }
+
+        return $violations;
+    }
```

PHPMD (2.15.0)

[warning] The method constructorValueReturns() has a Cyclomatic Complexity of 34. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)